features/works/work_create.feature

```diff
@@ -197,6 +197,29 @@ Feature: Create Works
       And I should see "Chapter"
       And I should see "1/?"
 
+  Scenario: Creating a new work in a series with some invalid things should not 500
```

Comment:
It's generally better to describe the correct behavior than the bug, so maybe something more like "should return you to the edit form, with the relevant error messages and the series information still filled in" instead of "should not 500"?